src/display/editor/stamp.js

```diff
@@ -170,9 +175,15 @@ class StampEditor extends AnnotationEditor {
         channels: data.length / (width * height),
       },
     });
-    if (!response || response.error || !response.output) {
+    if (!response || response.error) {
+      throw new Error("No valid response from the AI service.");
+    }
+    if (response.cancel) {
       return null;
     }
+    if (!response.output) {
+      throw new Error("No valid response from the AI service.");
```

Comment:
Would it make sense to have a *slightly* different error message here, to aid future debugging by being able to tell this case apart for the one just above here?